atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/atomic/TimestampExtractingAtomicTableTest.java

```diff
@@ -45,7 +45,7 @@ public void canExtractCommittedTransaction() throws ExecutionException, Interrup
         when(delegate.get(keys)).thenReturn(Futures.immediateFuture(commits));
 
         Map<Long, Long> expected = KeyedStream.of(keys).map(this::commitTs).collectToMap();
-        assertThat(timestampExtractingAtomicTable.get(keys).get()).isEqualTo(expected);
+        assertThat(timestampExtractingAtomicTable.get(keys).get()).containsExactlyInAnyOrderEntriesOf(expected);
```

Comment:
The javadoc for `get` doesn't contain any guarantee that the results are ordered the same way, so this is _probably_ fine, although `containsExactlyEntriesOf` would be stronger.